lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>Wording here is a little tricky, and speaks to a limitation of the current API --&gt; we're not particularly careful about coordinate systems here. Not sure if this is true for a decubitis orientation, for example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I don't think I got your point... Are you referring to this sentence? 

> <code>axes_reference</code> and <code>axes_evaluation</code> are tuples of numpy arrays, whose values define the (z, y, x) grid coordinates in the DICOM frame of reference.

Reply:
<p>I'll check the code, but I'm not sure that <code>zyx_and_dose_from_dataset()</code> actually returns a tuple of axes that are always strictly in the order <code>z, y, x</code>. For example, if the scan orientation is <em>Head First Decubitis Left</em>. If it's not, then we <em>could</em> try to be super specific about the wording (i.e., that for <em>this</em> case, "<em>the dose image is composed of...</em>"). But honestly, I wouldn't worry for now.</p>

Reply:
<p>I'm going to make the text more "generic" about this point then</p>